printer/downloadFile.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const fs = require('fs');
+const { v4: uuidv4 } = require('uuid');
+
+/**
+ * Download PDF file from URL.
+ * @param {Sring} url : PDF file url
+ * @returns Promise return true and the path to downloaded PDF file
+ * if it was successfully download. It then returns false if there
+ * was an error.
+ */
+async function downloadFileFromURL(url) {
+  return new Promise( ( resolve ) => {
+    axios({
+      url: url,
+      method: 'GET',
+      responseType: 'stream'
+    })
+      .then(response => {
+        const fileName = uuidv4();
+        const filePath = `/tmp/${fileName}.pdf`;
+        fs.writeFileSync(filePath, '');
+        const fileStream = fs.createWriteStream(filePath);
+        response.data.pipe(fileStream);
+        console.log('Downloaded file location: ', filePath);
```

Comment:
use logger here, not console.log, see [`tools/writeToSQS.js:1`](https://github.com/SCE-Development/Quasar/blob/831de7a7f3b07d23f2bb651992ee1c930d1e13a1/tools/writeToSQS.js#L1) as an example